airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -73,17 +77,13 @@
 public class BigQueryDestination extends BaseConnector implements Destination {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(BigQueryDestination.class);
-  private static final int MiB = 1024 * 1024;
-  static final String CONFIG_DATASET_ID = "dataset_id";
-  static final String CONFIG_PROJECT_ID = "project_id";
-  static final String CONFIG_DATASET_LOCATION = "dataset_location";
-  static final String CONFIG_CREDS = "credentials_json";
-  static final String BIG_QUERY_CLIENT_CHUNK_SIZE = "big_query_client_buffer_size_mb";
 
   private static final com.google.cloud.bigquery.Schema SCHEMA = com.google.cloud.bigquery.Schema.of(
       Field.of(JavaBaseConstants.COLUMN_NAME_AB_ID, StandardSQLTypeName.STRING),
-      Field.of(JavaBaseConstants.COLUMN_NAME_DATA, StandardSQLTypeName.STRING),
-      Field.of(JavaBaseConstants.COLUMN_NAME_EMITTED_AT, StandardSQLTypeName.TIMESTAMP));
+      // GCS works with only date\datetime formats, so need to have it a string for a while
+      // https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-csv#data_types
+      Field.of(JavaBaseConstants.COLUMN_NAME_EMITTED_AT, StandardSQLTypeName.STRING),
```

Comment:
The type of the column when creating the schema of the table was changed from timestamp to STRING

But when formatting the records to insert, they are still typed as timestamp?
https://github.com/airbytehq/airbyte/blob/888e9abee4b653054ae7fc80b860e966120f79b3/airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryRecordConsumer.java#L153

Reply:
ok so there was an issue for this here https://github.com/airbytehq/airbyte/issues/5959